app/src/main/java/com/github/se/polyfit/ui/screen/IngredientScreen.kt

```diff
@@ -109,6 +111,7 @@ private fun BottomBar(onClickAddIngred: () -> Unit, navigateForward: () -> Unit,
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 private fun TopBar(navigateBack: () -> Unit) {
```

Comment:
maybe out of scope, but, it may be better to just replace this composable with the simpletopbar to improve future maintainability

Reply:
Yeah something I noticed while going through the code. I saw a `TODO` somewhere about that. I agree with  you I will add it to the scrum board.